src/extension.ts

```diff
@@ -607,6 +607,31 @@ function launchMetals(
       }
     );
 
+    registerCommand(`metals.new-scala-worksheet`, async () => {
+      const sendRequest = (args: Array<string | undefined>) => {
+        return client.sendRequest(ExecuteCommandRequest.type, {
+          command: ServerCommands.NewScalaFile,
+          arguments: args,
+        });
+      };
+      const currentUri = window.activeTextEditor?.document.uri;
```

Comment:
I'm not sure I understand this. When would this return false?

Reply:
if there's no active text editor, I suppose. It can happen if (for instance) VSCode is open but there are no open files

Reply:
Yes, that's exactly the case.

Reply:
Sounds good, thanks!